### PR TITLE
Correct name of cypress.config in `build/build.php`

### PR DESCRIPTION
**Pull request by astridx**
`cypress.config.js` was removed from package in #39478
Then the file was renamed in https://github.com/joomla/joomla-cms/pull/39722

### CODE REVIEW

**Comment by wilsonge** [comment]
If this is in the beta release then we’ll need to add it to the delete files script too

**Comment by richard67** [comment]
> If this is in the beta release then we’ll need to add it to the delete files script too

@wilsonge Yes, even before beta we did that. But we do not bother authors of pull requests with that. Instead, I regularly do that before each release since alpha 1 :-)

File: build/build.php [hunk 1]
```diff
@@ -400,7 +400,7 @@ function clean_composer(string $dir)
     'composer.json',
     'composer.lock',
     'crowdin.yml',
-    'cypress.config.js',
+    'cypress.config.dist.js',
```
Comment (richard67) [inline build/build.php:403]: Please use same indentation as in the previous line.

Comment (richard67) [reply build/build.php:403]: Ah checked the file view. All ok. I was trapped by the small mobile diff view.

**Comment by richard67** [comment]
@wilsonge P.S.: Or did you mean the cleanup step in build.php so it does not go into packages?

**Comment by richard67** [comment]
It seems the preparation step for the system tests is failing in drone because cypress wants to use the file which with this PR is removed from the build. @Hackwar could you have a look on that? Does it need to change something in drone.xml, too?

**Comment by richard67** [comment]
Will update the branch, maybe this helps with drone.

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**